src/main/java/com/example/ibeproject/controller/FooterConfigurationController.java

```diff
@@ -0,0 +1,58 @@
+package com.example.ibeproject.controller;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.ibeproject.dto.footer.FooterConfigDTO;
+import com.example.ibeproject.service.FooterConfigService;
+
+@CrossOrigin(origins = "${cors.allowed.origin}")
```

Comment:
Can we define a separate config, where we can allow origins for all endpoints. Instead of defining everywhere

Reply:
Makes sense, will do.